src/Components/Api/Client.php

```diff
@@ -123,23 +130,34 @@ public function shops(AccessToken $token): array
     }
 
     /**
+     * @param AccessToken $token
      * @return License[]
+     * @throws \Psr\SimpleCache\InvalidArgumentException
```

Comment:
Please add throws only when it throwed in that method

Reply:
It's my IDE auto completion so I prefer to add it in the PHPDoc.
@throws removed in last commit.